samples/project-with-buildSrc/build.gradle.kts

```diff
@@ -1,2 +1,10 @@
 
 withHelloTask()
+
+tasks.create("printProfile") {
+    description = "Uses the extension property and the extension functions defined in buildSrc. Use with -Pprofile=prod."
+    group = "sample"
+
+    println("The current profile is $profile")
```

Comment:
:x: This should be moved to a `doLast {}` block

Reply:
Duh! Newbie mistake. I guess I'll never stop doing that one...